README.md

```diff
@@ -1 +1,5 @@
 # blogFrog
+Task for readme:
+Explain how to use this repo to new developers.
+
+We run this web app by running the app.py file with the command: python3 app.py
```

Comment:
it would be a better idea to format this better

Reply:
i'll have it formatted by the time i make the next PR